test/services/ConsignmentServiceSpec.scala

```diff
@@ -1,5 +1,6 @@
 package services
 
+import cats.implicits.catsSyntaxOptionId
```

Comment:
There is an used import: `import graphql.codegen.UpdateConsignmentSeriesId.{updateConsignmentSeriesId => ucs}`